templates/projects/presenter.html

```diff
@@ -66,6 +67,10 @@ <h3><a href="{{url_for('project.details', short_name=project.short_name)}}">{{ p
     <link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}">
 {% endassets %}
 
+{% if 'all' in config.CHAT_ENABLE or project.short_name in config.CHAT_ENABLE %}
+    {{ helper.chat(owner, current_user, project, n_tasks, num_expected_task_runs, num_remaining_task_runs, n_available_tasks_for_user, n_completed_tasks_by_user) }}
```

Comment:
I think before merge this PR will be good to create a PR on Pybossa pointing to this branch to ensure test are passing on the pybossa.

Reply:
Thank you, great idea! https://github.com/bloomberg/pybossa/pull/405